src/parser/Statement.ts

```diff
@@ -1141,7 +1141,7 @@ export class NamespaceStatement extends Statement implements TypedefProvider {
     ) {
         super();
         this.name = this.nameExpression.getName(ParseMode.BrighterScript);
-        this.symbolTable = new SymbolTable(parentSymbolTable);
+        this.symbolTable = new SymbolTable(parentSymbolTable, `Namespace ${keyword.text}`);
```

Comment:
`keyword.text` would contain "namespace". Did you mean `nameExpression.getName()`?